dropwizard-parent-pom/pom.xml

```diff
@@ -252,11 +251,6 @@
       </dependency>
       <dependency>
         <groupId>io.dropwizard</groupId>
-        <artifactId>dropwizard-spdy</artifactId>
```

Comment:
@eschuchmann The BOM approach sounds like it's going to drastically simplify this POM and make your work a lot easier.  I'm 👍  for going in this direction.  Don't think you'll have any issues with it.